script/Arcueid/Day 13/13_00_ARC13_4.txt

```diff
@@ -867,17 +867,17 @@ Still on her knees, Arcueid no longer even bothers to lift her head.
 [sha:7081ea035634bcb5264f7907abbdb3d281849b2d]{
 -- Page 357, Offset 32657.
 -- 「“無意味だった。はじめから無意味だった。
-"It's meaningless. It's been meaningless from the start.
+"It was meaningless. It has always been meaningless.
 }
 [sha:025b25b518165b9c03e5ead6cd54099e6e634653]{
 -- Page 357, Offset 32658.
 -- あなたの<航|た><海|び>は、八百年前から無意味だった。
-Your voyage through life has been meaningless for the past 800 years.
+Your voyage through life has been meaningless ever since those 800 years ago.
```

Comment:
Not really grammatical here, I'd revert.

Reply:
True, lost myself there a bit:

"ever since that time 800 years ago."

The point of the sentence is to call back to Roa and Arcueid's meeting, so I feel rather than just saying "the past 800 years", harkening to what happened 800 years ago is more important.

Reply:
Yeah `ever since that day 800 years ago` works for me